sections/edit-password-form.liquid

```diff
@@ -56,104 +73,83 @@
   style="{{- variables | escape -}}"
 >
   <div class="account__container container">
-    <h3 class="account__title text-center">
-      {%- if title != blank -%}
-        {{- title -}}
-      {%- else -%}
-        Change your password
-      {%- endif -%}
-    </h3>
+    <div class="account__inner{% if show_border %} account__inner--border{%- endif -%}">
+      <h3 class="account__title text-center">
+        {{- title | default: "Change your password" -}}
+      </h3>
 
-    {%- if description != blank -%}
-      <div class="account__divider">
-        {{- description -}}
-      </div>
-    {%- endif -%}
+      {%- if description != blank -%}
+        <div class="account__divider text-center">
+          {{- description -}}
+        </div>
+      {%- endif -%}
 
-    {%- form 'edit_password' -%}
-      <div class="account-fieldset__block account__divider--small{%- if form.errors.password -%}account-fieldset--error{%- endif -%}">
-        <label for="user_password" class="account-fieldset__label">
-          {%- if password_label != blank -%}
-            {{- password_label -}}
-          {%- else -%}
-            New password
-          {%- endif -%}
+      <hr class="account__separator">
 
-          <span class="account-fieldset__label-backlight">
-            {%- if password_label != blank -%}
-              {{- password_label -}}
-            {%- else -%}
-              New password
-            {%- endif -%}
-          </span>
-        </label>
+      {%- form 'edit_password' -%}
+        <div class="account-fieldset__block account__divider--small{%- if form.errors.password -%} account-fieldset--error{%- endif -%}">
+          <label for="user_password" class="account-fieldset__label">
+            {{- password_label | default: "New password" | append: "*" -}}
+          </label>
 
-        <input
-          value="{{- form.password -}}"
-          placeholder=" "
-          class="account-fieldset__input"
-          autofocus="autofocus"
-          autocomplete="new-password"
-          type="password"
-          name="user[password]"
-          id="user_password">
+          <input
+            value="{{- form.password -}}"
+            placeholder=" "
```

Comment:
Do we need a placeholder at all here?

Reply:
You're right, it isn't necessary in Impact